build.gradle

```diff
@@ -47,7 +116,8 @@ processResources {
 }
 
 tasks.withType(JavaCompile).configureEach {
-    // Loom requires J17.
+    // Minecraft 1.18 (1.18-pre2) upwards uses Java 17.
```

Comment:
sounds about right

Reply:
Lol. I had a hard time building the project so I copied over my build.gradle files. Feel free to keep it or remove it 👍